src/architecture/toolSystem.js

```
@@ -0,0 +1,389 @@
+        description: 'Perform mathematical calculations',
+        parameters: {
+            expression: {
+                type: 'string',
+                required: true,
+                description: 'Mathematical expression to evaluate'
+            }
+        },
+        execute: async (parameters) => {
+            try {
+                // Safe evaluation of mathematical expressions
+                const result = eval(parameters.expression.replace(/[^0-9+\-*/().]/g, ''));
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)